eeps/eep-0048.md

```diff
@@ -152,9 +155,11 @@ For each entry in `Docs`, we have:
     may be rendered as as a single line or two lines. It exists
     exclusively for exhibition purposes
 
-  * `Doc` - a binary with the documentation or the atom `none`
-    in case there is no documentation or the atom `hidden` if
-    documentation has been explicitly disabled for this entry
+  * `Doc` - a map with the documentation language as key, such as
+    `<<"en">>` or `<<"pt_BR">>`, and the documentation as a binary value.
```

Comment:
You have somehow managed to get two non-breaking-space in the beginning of this line: [32,160,32,160]